tests/Fixer/Basic/CurlyBracesPositionFixerTest.php

```diff
@@ -312,35 +312,6 @@ public static function provideFixCases(): iterable
             ['control_structures_opening_brace' => 'next_line_unless_newline_at_signature_end'],
         ];
 
-        yield 'try catch finally (default)' => [
```

Comment:
Similar like in `BracesPositionFixerTest`, let's modify these cases to actually test `try/catch/finally`.